ocs_ci/utility/networksplit/network-split.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
```

Comment:
What about to use:
`set -xv`
It's is always better for debugging purpose to see this debug output in shell script.

Reply:
This is not necessary in this case.